src/components/grid-aware/Footer/InfoBlock/InfoBlock.module.css

```
@@ -0,0 +1,44 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.infoBlock {
+  background-color: var(--color-midnight-blue);
+  color: var(--color-gray-300);
```

I think in the designs, this is actually Gray 400 (`#e0e0e0`).

```suggestion
  color: var(--color-gray-400);
```

---

ok